src/operator/controllers/intents_reconcilers/generate.go

```diff
@@ -3,5 +3,5 @@ package intents_reconcilers
 //go:generate go run go.uber.org/mock/mockgen@v0.2.0 -destination=./mocks/mock_k8s_client.go -package=intentsreconcilersmocks sigs.k8s.io/controller-runtime/pkg/client Client
 //go:generate go run go.uber.org/mock/mockgen@v0.2.0 -destination=./mocks/mock_istio_manager.go -package=intentsreconcilersmocks -source=../istiopolicy/policy_manager.go PolicyManager
 //go:generate go run go.uber.org/mock/mockgen@v0.2.0 -destination=./mocks/mock_service_resolver.go -package=intentsreconcilersmocks -source=../../../shared/serviceidresolver/serviceidresolver.go ServiceResolver
-//go:generate go run go.uber.org/mock/mockgen@v0.2.0 -destination=./mocks/mock_external_netpol_handler.go -package=intentsreconcilersmocks -source=./network_policy.go externalNetpolandler
+//go:generate go run go.uber.org/mock/mockgen@v0.2.0 -destination=./mocks/mock_external_netpol_handler.go -package=intentsreconcilersmocks github.com/otterize/intents-operator/src/operator/controllers/intents_reconcilers/networkpolicy ExternalNetpolHandler
```

Comment:
Because I want to generate this specific interface, and it's only possible with the absolute path mode.
https://stackoverflow.com/questions/73012646/why-mockgen-generates-all-interfaces-in-file